torchgeo/datasets/airphen.py

```diff
@@ -0,0 +1,92 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+"""Airphen dataset."""
+
+from typing import Any, Optional
+
+import matplotlib.pyplot as plt
+from matplotlib.figure import Figure
+
+from .geo import RasterDataset
+from .utils import RGBBandsMissingError, percentile_normalization
+
+
+class Airphen(RasterDataset):
+    """Airphen dataset.
+
+    `Airphen <https://ideol.sakura.ne.jp/img/20170123_HiphenAirphenKeyfeatures.pdf>`__
+    is a multispectral scientific camera developed by agronomists and photonics
+    engineers at `Hiphen <https://www.hiphen-plant.com/>`_ to match plant measurements
+    needs and constraints. Its high flexibility, ease of use and radiometric quality
```

Comment:
This description / main characteristics / key features sounds like it is copy pasted from marketing material somewhere, e.g. "Powerful analysis". Can you trim this down or remove? If someone is using this dataset, then they should know about the main characteristics / key features

Reply:
> sounds like it is copy pasted from marketing material somewhere

You are 100% correct. The only "documentation" I could find about this sensor is marketing material. Let me do some trimming...